files/master.cfg

```diff
@@ -31,12 +30,22 @@ class WorkerAdmins:
         except KeyError:
             return None
 
+{% endraw %}
+# Configurations controlled by ansible
+
+cfg_title = '{{ afsbotcfg_title }}'                 # Title for buildbot banner
```

Comment:
If you use a class or attribute dictionary, you don't have to change the rest of this file, and it would look much nicer and less like c-language in python.

Reply:
ack.. I'll just use a settings = { ... }